packages/blade/src/components/Amount/Amount.tsx

```diff
@@ -154,6 +159,7 @@ const addCommas = (amountValue: number, currency: Currency): string => {
   return amountValue.toLocaleString(locale);
 };
 
+// This function returns the humanized amount ie: human readble currency format
```

Comment:
An example might be more helpful on what we mean by "human readable". Also you can use the following comment style so it shows in vscode hints:

```js
/**
 * This function ...
 *
 */
```